src/styles/utilities.css

```
@@ -61,6 +61,68 @@
+    line-height: var(--type-heading2-leading);
+  }
+  @media (max-width: 639px) {
+    .font-heading2-rsp {
+      font-size: var(--type-heading3-size);
+      font-weight: var(--type-heading3-weight);
+      line-height: var(--type-heading3-leading);
+    }
+  }
+
+  /* 모바일(< 640px)에서 그리드를 1열로 강제 — tablet: 커스텀 변형 우선순위 우회 */
+  .grid-mobile-1 {}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)